[0m02:29:24.824902 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000217A358DE50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000217A54B3200>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000217A5E1E720>]}


============================== 02:29:24.832902 | 0d218339-3a02-4032-9084-76e4356a0e3b ==============================
[0m02:29:24.832902 [info ] [MainThread]: Running with dbt=1.8.6
[0m02:29:24.832902 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\eobay\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt run', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m02:29:24.840907 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\eobay\OneDrive\Desktop\Projects\Y-Combinator_datapipline\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m02:29:24.840907 [debug] [MainThread]: Command `dbt run` failed at 02:29:24.840907 after 0.24 seconds
[0m02:29:24.840907 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000217A5DD1C40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000217A5CEAE70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000217A358DE50>]}
[0m02:29:24.840907 [debug] [MainThread]: Flushing usage events
[0m13:37:54.041990 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B33B2A7530>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B33AF5A960>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B337A2D700>]}


============================== 13:37:54.049949 | 25374613-255b-4a7e-84a9-9883dcc60cc8 ==============================
[0m13:37:54.049949 [info ] [MainThread]: Running with dbt=1.8.6
[0m13:37:54.051948 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\eobay\\.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt run', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m13:37:54.051948 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\eobay\OneDrive\Desktop\Projects\Y-Combinator_datapipline\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m13:37:54.053949 [debug] [MainThread]: Command `dbt run` failed at 13:37:54.053949 after 0.19 seconds
[0m13:37:54.054950 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B33ABC7380>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B33B0998E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B33B152DB0>]}
[0m13:37:54.055949 [debug] [MainThread]: Flushing usage events
